README.md

```diff
@@ -2,15 +2,15 @@ This is a user-space file system for interacting with [Google Cloud
 Storage][gcs]. It is not yet ready for public consumption, so please do not use
 it for anything of importance in its current state.
 
-[gcs]: https://cloud.google.com/storage/
+[gcs]: https://cloud.google.com/storage/docs
```

Comment:
Is this really a better URL? The other one is a nice product overview, etc. It's clearly the landing page for GCS.